praw/models/reddit/submission.py

```diff
@@ -1,6 +1,7 @@
 """Provide the Submission class."""
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
 from urllib.parse import urljoin
+from logging import getLogger
```

Comment:
```suggestion
from warnings import warn
```
Can we use `warnings.warn` instead here?